Gemfile.lock

```diff
@@ -0,0 +1,210 @@
+GEM
```

Comment:
@timoschilling What's your concern? This obviously needs more maintenance but I thinks it's definitely less trouble than continuously broken builds because of 3rd party dependency changes. I strongly believe it is a good idea to enforce environment consistency between all developers and CI.